gooey/gui/controller.py

```diff
@@ -108,8 +171,8 @@ def required_section_complete(self):
   def success_dialog(self):
     self.show_dialog(i18n._("execution_finished"), i18n._('success_message'), wx.ICON_INFORMATION)
 
-  def error_dialog(self, error_msg):
-    self.show_dialog(i18n._('error_title'), i18n._('uh_oh').format(error_msg), wx.ICON_ERROR)

```

Comment:
@Shura1oplot Hmm... this is interesting. I didn't know it was common to pump non error messages over stderr. You're exposing my noobish level of linux knowledge! ; )  

After spending a few minutes reading up on the behavior, I yield to your explanation. I can see why programs would use stderr for things other than a simple error stream. 

So... as you said, it seems Gooey's original behavior was indeed broken. 

Good fix!